code/src/UI/Services/ProjectConfigInfoService.cs

```diff
@@ -229,17 +243,17 @@ private bool IsSplitView(string platform)
                     if (IsCSharpProject())
                     {
                         // Prism doesn't have an activation service, but will have PageToken constants
-                        return ExistsFileInProjectPath("Views", "ShellPage.xaml")
-                            && (ExistsFileInProjectPath("Services", "ActivationService.cs") || ExistsFileInProjectPath("Constants", "PageTokens.cs"));
+                        return ExistsFileInProjectPath("ShellPage.xaml", "Views")
+                            && (ExistsFileInProjectPath("ActivationService.cs", "Services") || ExistsFileInProjectPath("Constants", "PageTokens.cs"));
```

Comment:
ExistsFileInProjectPath("Constants", "PageTokens.cs") should be ExistsFileInProjectPath("PageTokens.cs", "Constants")